jobs/azure_cpi/templates/cpi.json.erb

```diff
@@ -39,11 +37,22 @@
     params['cloud']['properties']['azure']['location'] = location
   end
 
-  if_p('azure.client_secret') do |client_secret|
-    params['cloud']['properties']['azure']['client_secret'] = client_secret
-  end.else do
-    certificate = p('azure.certificate', '')
-    raise 'both "client_secret" and "certificate" are not provided. Please specify at least one of them.' if certificate.empty?
+  if p('azure.managed_service_identity.enabled')
+    params['cloud']['properties']['azure']['managed_service_identity'] = {
+      "enabled" => true
+    }
+  else
```

Comment:
extra empty after else.

Reply:
There's no extra empty line. See the line number.